CHANGELOG.md

```diff
@@ -13,5 +13,6 @@ CHANGELOG
 * Centralise the work for Autonaming in providers.
 * Avoid setting conflicting default values ([91](https://github.com/pulumi/pulumi-terraform-bridge/pull/91)).
 * Require explict C# namespaces.
+* Remove support for synchronous data sources.
```

Comment:
Just to make things easier for keeping this branch up to date - should we be finding a way to add these that doesn't lead to merge conflicts?  Perhaps a dedicated `2.0` section at the top might avoid conflicts?

Reply:
wfm